.github/workflows/main.yml

```diff
@@ -31,15 +33,11 @@ jobs:
       with:
         python-version: '3.x'
 
-    # needed because apt version is too old
-    - name: Install Build Tools (pip)
-      run: pip install meson ninja
-
     - name: Install Build Tools (apt)
-      run: sudo apt install cmake build-essential libsdl2-dev libsdl2-mixer-dev libogg-dev libopusfile-dev libpng-dev libzip-dev libx11-dev libwayland-dev imagemagick python3-docutils libwebp-dev
+      run: sudo apt install meson cmake build-essential libsdl2-dev libogg-dev libopusfile-dev libpng-dev libzip-dev libx11-dev libwayland-dev imagemagick python3-docutils libwebp-dev
```

Comment:
cmake shouldn't be needed either, missed it last time.

Reply:
Or imagemagick. It's only used for development-specific tasks like atlas generation, but not during build.